etc/input.conf

```diff
@@ -143,6 +143,7 @@
 #L cycle-values loop-file "inf" "no"    # toggle infinite looping
 #ctrl+c quit 4
 #DEL script-binding osc/visibility      # cycle OSC display
+#ctrl+h cycle-values hwdec "auto" "no"  # cycle hardware decoding
```

Comment:
Maybe also "auto-copy"?

Reply:
Would that be useful?

Reply:
Might help with gpu-apis/context that don't support many non-copy hwdecs while decoding high framerate hevc/vp9.

Reply:
I don't see much of a point. If direct hwdec is not supported, auto should select a -copy variant too.